applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -40,10 +40,9 @@ class MechanicalSolver(object):
     def __init__(self, main_model_part, custom_settings):
         default_settings = KratosMultiphysics.Parameters("""
         {
-            "solver_type": "structural_mechanics_solver",
+            "solver_type": "Static",
             "echo_level": 0,
```

Comment:
I will not insist on this, but I prefer using the python script name in "solver_type" compared to custom names like Static, Dynamics, etc. My argument for this is that if you use the script name:

1) You don't need to search in the code for which python solver file gets loaded. It tells you directly without some intermediate keyword.

2) The MainKratos.py doesn't need to be modified since you tell it directly what solver to load. Everything related to the analysis type is left to that solver. With keywords like Static, the MainKratos.py already needs to know that this is a structural analysis and check for the solver to load.